packages/onboarding/src/hooks/use-persisted-state.ts

```diff
@@ -23,6 +23,20 @@ function getPersistedState( key: string, storage: Storage, TTL: number ) {
 	}
 }
 
+/**
+ * Clears all persisted state created by useStepPersistedState
+ * @param storage The storage to clear (defaults to localStorage)
+ */
+function clearStepPersistedState( storage: Storage = localStorage ): void {
+	const keys = Object.keys( storage );
+	const persistedKeys = keys.filter( ( key ) => key.startsWith( `${ VERSION }-${ KEY }` ) );
```

Comment:
If we we're feeling real fancy we could maybe remove all keys prefixed by `${VERSION}-${KEY}-${flow}`? Restarting a flow should only reset the data for that flow maybe?